src/main/resources/application.properties

```diff
@@ -3,6 +3,7 @@ logging.level.io.swagger.models.parameters.AbstractSerializableParameter=ERROR
 spring.jackson.serialization.fail-on-empty-beans=false
 server.servlet.context-path=/api
 server.port=${PORT:8080}
+endpoints.cors.allowed-origins=http://localhost:3000, https://dokazovi-fe.herokuapp.com
```

Comment:
Please try to use it with the injection variable:
endpoints.cors={ALLOWED_ORIGIN: http://localhost:3000, https://dokazovi-fe.herokuapp.com}

Reply:
I have the same error as you do in this case

Reply:
My version works

Reply:
Also we were talking about array of strings, so please investigate if we need it

Reply:
But I use this one - "private String [] allowedOrigins".  Did I misunderstand something?

Reply:
I just don't see split function